.ci/aarch64/clean_up_aarch64.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 ARM Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+stale_process_union=( "docker-containerd-shim" "qemu-system-aarch64" )
```

Comment:
You might want to consider adding the `kata-shim` and `kata-proxy` here as well.
I guess you've added what you have seen 'left around' so far - but we can probably pre-empt some other components that should not be present but might :-)